common/src/main/java/io/netty/util/internal/StringUtil.java

```diff
@@ -48,19 +48,9 @@
 
     static {
         // Generate the lookup table that converts a byte into a 2-digit hexadecimal integer.
-        int i;
-        for (i = 0; i < 10; i++) {
-            BYTE2HEX_PAD[i] = "0" + i;
-            BYTE2HEX_NOPAD[i] = String.valueOf(i);
-        }
-        for (; i < 16; i++) {
-            char c = (char) ('a' + i - 10);
-            BYTE2HEX_PAD[i] = "0" + c;
-            BYTE2HEX_NOPAD[i] = String.valueOf(c);
-        }
-        for (; i < BYTE2HEX_PAD.length; i++) {
+        for (int i = 0; i < BYTE2HEX_PAD.length; i++) {
```

Comment:
Simplification in this code block is done instead of a just correction `('a' + i - 10) -> ('a' + (i - 10))` which relates for current PR.

Reply:
@fenik17 got it... that said I think it not really matters here as its in a static block.